i18n/en.json

```diff
@@ -2023,45 +2027,209 @@
     "translation": "Thank you for contacting Mattermost. We will follow up with you soon."
   },
   {
-    "id": "api.server.warn_metric.bot_response.number_of_users.mailto_body",
-    "translation": "Mattermost Contact Us request. My team has now {{.Limit}} users and I am considering Mattermost Enterprise Edition.\r\n"
+    "id": "api.server.warn_metric.bot_response.start_trial_failure.message",
+    "translation": "Trial license could not be retrieved. Visit https://mattermost.com/trial/ to request a license."
   },
   {
     "id": "api.server.warn_metric.contact_us",
-    "translation": "Acknowledge"
+    "translation": "Contact Us"
   },
   {
     "id": "api.server.warn_metric.contacting_us",
-    "translation": "Acknowledging"
+    "translation": "Contacting Us"
+  },
+  {
+    "id": "api.server.warn_metric.email_domain.contact_us.email_body",
+    "translation": "Mattermost contact us request. I'm interested in learning more about using Guest Accounts.\r\n"
+  },
+  {
+    "id": "api.server.warn_metric.email_domain.notification_body",
+    "translation": "Projects often involve people both inside and outside of your organization. With Guest Accounts, you can bring external partners into your Mattermost system and specify who they can work with and what they can see.\r\n\r\n[Learn more about enabling Guest Accounts](https://www.mattermost.com/docs-guest-accounts/?utm_medium=product&utm_source=mattermost-advisor-bot&utm_content=guest-accounts).\r\n\r\nBy clicking Contact Us, you'll be sharing your information with Mattermost, Inc., to learn more about upgrading. [Learn more](https://mattermost.com/pl/default-admin-advisory)"
```

Comment:
re: @esadur and @jasonblais

Reply:
@mkraft @catalintomai I'm fine with this suggestion, though I want to keep the "Inc." So it would read "By clicking Contact Us, you'll be sharing your information with Mattermost, Inc." I am changing this in the 'Admin Advisor - Text Copy' doc

Reply:
@catalintomai this needs to be changed here too. Please update it to:

`By clicking Contact Us, you'll be sharing your information with Mattermost, Inc.` instead of `By clicking Contact Us, you'll be sharing your information with Mattermost, Inc., to learn more about upgrading`